docs/change-log/index.md

```diff
@@ -1,6 +1,8 @@
 (change-log)=
 # Change log
+- [v1.3.1](#v1-3-1)
 - [v1.3.0](#v1-3-0)
+- [v1.2.3](#v1-2-3)
```

Comment:
missing 1.2.2

Reply:
1.2.2 was not delivered to any customer - no need to include it

Reply:
This is a full changelog and should include versions that we release, not versions being used atm